tox.ini

```diff
@@ -63,15 +60,8 @@ deps =
 commands =
     yamllint .
 
-[testenv:py38]
-description = Python 3.8 unit tests [Mandatory]
-commands =
-    {[testenv]pytest_command}
-deps = 
-    -rrequirements-test.txt
-
-[testenv:py39]
-description = Python 3.9 unit tests [Mandatory]
+[testenv:py312]
```

Comment:
same here? 

Reason: So it's easy to drop 3.12 when the time comes and we can rely on 3.13 in the meanwhile

Reply:
I've added tests for 3.13, please check.